src/WebGPU/Silk.NET.WebGPU/WebGPULibraryNameContainer.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Silk.NET.Core.Loader;
+
+namespace Silk.NET.WebGPU
+{
+    /// <summary>
+    /// Contains the library name of webgpu.
+    /// </summary>
+    internal class WebGPULibraryNameContainer : SearchPathContainer
```

Comment:
How are we going to manage WGPU vs Dawn? Maybe something like what we did in Silk.NET.OpenAL?

Reply:
I was just going to assume 'you rename the lib', but ill check what OpenAL does

Reply:
Im unable to find a single source of all the library names that dawn builds with, so perhaps we can have some documentation about lib names (or maybe once we package our own wgpu and dawn libs we can just rename it ourself?)

Reply:
Raised #1137